packages/payer-logo/src/PayerLogo.stories.tsx

```diff
@@ -9,6 +10,9 @@ export default {
   parameters: {
     docs: {
       // page: README,
+      description: {
+        component: "Easy to use component to display the payer's logo given the payer's ID",
```

Comment:
just to really emphasize the state of the package, I'd either start or end the description with it's DEPRECATED status.

Reply:
done